reproject/hips/core.py

```diff
@@ -106,8 +116,14 @@ def image_to_hips(
     if progress_bar is None:
         progress_bar = lambda x: x
 
+    if level is None:
+        level = determine_healpix_level(wcs_in, tile_size)
+        pixel_size = (4 * np.pi * u.sr / (12 * (2**level)**2)).to(u.arcsec**2)**0.5
+        tile_angular_size = pixel_size * tile_size
+        logger.info(f"Automatically set the HEALPIX level to {level} with tile size {tile_angular_size} and pixel size {pixel_size}")
+
     # Create output directory (and error if it already exists)
-    os.makedirs(output_directory, exist_ok=False)
+    os.makedirs(output_directory, exist_ok=overwrite)
```

Comment:
So one issue here is that it won't empty the existing directory, it will just add tiles to it. Ideally, if overwrite=True, we should probably remove the output directory, right?

Reply:
I think the idea is that we _do_ want to add tiles to existing directories - at least, i think that's what aladin does.  But I can very easily be convinced that we should just delete the whole directory

Reply:
I think it would mislead people into thinking it was co-adding the frames, which it wouldn't be though.